osquery/core/tests/conversions_tests.cpp

```diff
@@ -55,7 +55,18 @@ TEST_F(ConversionsTests, test_utc) {
   FLAGS_utc = true;
   auto t2 = getUnixTime();
   auto rt2 = std::time(nullptr);
-  auto at2 = std::mktime(std::gmtime(&rt2));
+
+#ifdef WIN32
```

Comment:
If we need to do platform-specific stuff and there are no platform-API specific calls then use the `isPlatform(TYPE_WINDOWS)` or otherwise. This allows easier static analysis and code coverage.